test/infrastructure/docker/controllers/alias.go

```diff
@@ -72,6 +72,24 @@ func (r *DockerClusterReconciler) SetupWithManager(ctx context.Context, mgr ctrl
 	}).SetupWithManager(ctx, mgr, options)
 }
```

Comment:
Sorry. We have to implement everything you did for DockerMachineTemplate also for DevMachineTemplate (I would simply copy&paste it)